urllib3/util/ssl_.py

```diff
@@ -195,6 +197,16 @@ def create_urllib3_context(ssl_version=None, cert_reqs=ssl.CERT_REQUIRED,
     """
     context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
 
+    if hasattr(context, 'urllib3_wrapper_context'):
```

Comment:
Instead of doing this hasattr for a special attribute, would it not make sense to just move this warning to the mock ``SSLContext().wrap_socket()``?